service/switchboardapp/src/main/java/com/switchboard/app/domain/Device.java

```diff
@@ -0,0 +1,30 @@
+package com.switchboard.app.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.*;
+
+import javax.persistence.*;
+
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter@Setter
+@ToString
+@JsonIgnoreProperties({"hibernateLazyIntializer","handler","encoder","decoder"})
+public class Device {
+
+    @Id
+    @Column(name = "serial_number")
+    private Long serialNumber;
+
+    @Column(name = "display_name")
+    private String DisplayName;
```

Comment:
Add @NotNull to serial number and display name

Reply:
fixed